packages/common/container-definitions/api-extractor/api-extractor.legacy.json

```diff
@@ -1,4 +1,8 @@
 {
 	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
-	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.legacy.json"
+	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.legacy.json",
+	"apiReport": {
+		"enabled": true,
+		"reportVariants": ["alpha", "beta"]
```

Comment:
Another one where I was expecting to see beta and public, instead of alpha and beta.

Reply:
Removed